plugins/languages/typescript-tools.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, pkgs, helpers, config, ... }:
+with lib;
+let cfg = config.plugins.typescript-tools;
+in {
+  extraPlugins = [ cfg.package ];
+  options.plugins.typescript-tools = {
```

Comment:
```suggestion
in {
  options.plugins.typescript-tools = {
```

Reply:
I don't know what's up with nil_ls, but it looooves to butcher this repo's formatting. As of now I'm just rolling with it and running `nix fmt` afterwards. :sob:
If you know of something I should incorporate into my nvim setup, please let me know because this is ANNOYING

Reply:
Don't forget to remove this wrong line (`extraPlugins = [ cfg.package ];`)